src/pages/UsersPage/UsersPage.tsx

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from 'react-redux';
+import { selectAllUsers } from 'store/redux/users/selectors';
+import { UsersPageCard, UsersPageContainer, UsersPageFlex, UsersPageH2, UsersPageP } from "./styles"
+
+function UsersPage() {
+  const users = useSelector(selectAllUsers);
+
+  return (
+    <UsersPageContainer>
+      <UsersPageH2>Users Page</UsersPageH2>
+      <UsersPageFlex>
+        {users.map((user, index) => (
```

Comment:
не очень хорошо использовать index  в качестве key
вот что писали в библиотеке: "We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state."
Вот ссылка: https://legacy.reactjs.org/docs/lists-and-keys.html